### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery()

### ISSUE THREAD

**Issue by herrbischoff** [kisenka/svg-sprite-loader#78]
I just added the configuration to webpack, it is working fine but I always get this warning:

```
...s/svg-sprite-loader/lib/web/sprite.js(node:36437) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
Hash: ffb4fa842d0159348c88
Version: webpack 2.2.1
Time: 3725ms
```

Am I doing something wrong or does `svg-sprite-loader` need to be updated to use `getOptions()`?

**Comment by jeremyzahner** [comment]
@kisenka Is this issue something one could support you with? Meaning, if i fix it in a PR, will you consider merging it?

**Comment by kisenka** [comment]
@jeremyzahner sure, PR is always welcome

**Comment by kisenka** [comment]
@herrbischoff @jeremyzahner
Should be fixed in upcoming 2.0 version. Please read about it here #91 and take part in discussion/voting.

**Comment by jeremyzahner** [comment]
@kisenka Nice!

**Comment by kisenka** [comment]
Fixed in 2.0.1, please update.

**Comment by herrbischoff** [comment]
Thank you! I will definitely try the new version next chance I get.
